docs/superpowers/plans/2026-05-01-calc-column-layout.md

```
@@ -0,0 +1,554 @@
+              slotErrors={slotErrors}
+            />
+          );
+        })}
+      </div>
+    </div>
+  );
+}
+```
+
+- [ ] **Typecheck:**
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)